### DISCUSSION TITLE
Which flavor of regex GitVersion uses?

### DISCUSSION THREAD

**Discussion by iisti** [GitTools/GitVersion#4026] [Q&A] [answered]
One can determine branches with regex. I wonder which flavor of regex GitVersion uses?

Example of the regex configuration:
```
branches:
  develop:
    regex: ^dev(elop)?(ment)?$
```

**Comment by ChadMcCaffery** [comment, marked as answer]
GitVersion uses this one:
[Regular Expression Language - Quick Reference | Learn .NET](https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference)
